code/game/objects/items/weapons/implants/implant_traitor.dm

```diff
@@ -17,64 +17,70 @@
 	return dat
 
 /obj/item/implant/traitor/implant(mob/M, mob/user)
+
+	if(!user.mind.has_antag_datum(/datum/antagonist))  // if the user isnt an antagonist at all, don't let them use it
```

Comment:
I'd rather have mindslave be usable by non-traitor.

Reply:
Will remove that part then